src/vm/methodtablebuilder.cpp

```diff
@@ -10257,6 +10278,16 @@ void MethodTableBuilder::CheckForSystemTypes()
         {
             pMT->SetIsNullable();
         }
+#ifdef FEATURE_SPAN_OF_T
+        else if (strcmp(name, g_SpanName) == 0)
+        {
+            pMT->SetIsByRefLike();
+        }
+        else if (strcmp(name, g_ReadOnlySpanName) == 0)
+        {
+            pMT->SetIsByRefLike();
+        }
```

Comment:
Is this going to cause any problems if someone has their own ```System.Span<T>``` or ```System.ReadOnlySpan<T>``` defined in their own library?